### ISSUE TITLE
Login failed

### ISSUE THREAD

**Issue by jwilk** [jwilk/mbank-cli#71]
I'm having frequent login problems:
```
mbank-cli: login failed: Przykro nam, usługa jest w tej chwili niedostępna. Prosimy spróbować ponownie. Jeśli błąd się powtórzy prosimy o kontakt z Centrum Obsługi Telefonicznej...
```
(Before f5c1f3fecfaa71d95ea5c36c9b6206d5b3a6b46c the error message was less helpful.)
This doesn't happen all the time. It looks like the first login always fails, but the subsequent one always succeeds. (And then it fails again after the session expires...)
I don't know what's going on here. :-/

**Comment by jwilk** [comment]
> *Przykro nam, usługa jest w tej chwili niedostępna. Prosimy spróbować ponownie. Jeśli błąd się powtórzy prosimy o kontakt z Centrum Obsługi Telefonicznej...*

In English that is:

*Sorry, the service is currently unavailable. Please try again. If the error persists, please contact the Call Center...*

**Comment by jwilk** [comment]
To be clear, I haven't seen anything like that when using the web UI.

**Comment by jwilk** [comment]
> It looks like the first login always fails, but the subsequent one always succeeds.

Actually logging in appears to fail, but when you try again you're already logged in huh.

55600a5931f38acf773b100ad93518b37ff4a8a4 fixes it for me.

**Comment by jwilk** [comment]
Fixed in [1.8](https://github.com/jwilk/mbank-cli/releases/tag/1.8).
